recipe/meta.yaml

```diff
@@ -1,23 +1,30 @@
 {% set name = "swmm-toolkit" %}
-{% set version = "0.15.2" %}
+{% set version = "0.16.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/swmm-toolkit-{{ version }}.tar.gz
-  sha256: bc3d36e6cca656a6d18c4077b9fc429b54b65f8f7f9091dac5f5b7cd5a66f6dc
+  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name | replace('-', '_') }}-{{ version }}.tar.gz
+    sha256: ae213a1b4a0a3c841061bdaf20d74ee37dd45de83eded2d5314362b488066568
+  - git_url: https://github.com/pyswmm/Stormwater-Management-Model
+    git_rev: 24cdf73c3d7cf3bb62f0bff150d5987f1f826889
+    folder: swmm-solver
 
 build:
   script:
+    - set NO_STABLE_ABI=true  # [win and py>=313]
+    - export NO_STABLE_ABI=true  # [unix and py>=313]
```

Comment:
Why are you only disabling stable abi for py>=313? There is no reason to build stable ABI if conda is distributing wheels for each version of python separately anyway. That, and you're leaving some performance gains on the table for python versions <3.13 by not using python version specific optimizations.